cmd/scip/lint.go

```diff
@@ -236,6 +240,22 @@ func (st *symbolTable) addOccurrence(path string, occ *scip.Occurrence) error {
 	return nil
 }
 
+func lintSymbolString(symbol string, context string) error {
+	if symbol == "" {
+		return emptyStringError{what: "symbol", context: context}
+	}
+	sym, err := scip.ParseSymbol(symbol)
+	if err != nil {
+		// TODO: This should be linted (but it makes all the tests fail)
```

Comment:
Writing the long symbol names makes writing tests tedious, ideally we'd have a config object to disable some checks in tests. Maybe you can use a global variable here? It'd be off by default, and only one test would set it when checking for this code path. The other tests wouldn't need to worry about producing grammatically correct symbol names.

Reply:
What about introducing a constructor function for the tests? Something like `sym("a")` that expands to `. . . . a#`?

Reply:
Maybe we can move that logic to `makeIndex` or `parseSymbolInfo`? In hindsight, I'm not sure if adding a DSL to specify relationships was a good idea, but I didn't want to have large chains of functions/methods when writing test cases. 🙈

Reply:
Okay I tried... but the DSL was a bit too much too handle so I went with the global as you suggested :D